pkg/kubelet/config/file.go

```diff
@@ -138,23 +138,23 @@ func (s *sourceFile) extractFromDir(name string) ([]*v1.Pod, error) {
 
 		switch {
 		case statInfo.Mode().IsDir():
-			glog.V(1).Infof("Not recursing into config path %q", path)
+			glog.V(1).Infof("Not recursing into manifest path %q", path)
```

Comment:
Should this be logged as an error? Sounds like something that would happen if someone didn't provision the manifest path correctly.